src/site/content/en/blog/wakelock/index.md

```diff
@@ -0,0 +1,263 @@
+addition, you also need a way to abort the wake lock, which works through the
+generic `AbortController` interface. Therefore, you first create a new
+[`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController),
+and then pass the controller's
+[`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
+as the second parameter to `WakeLock.request()`. Two things can happen next
+that you need to `catch`:
+
+* The wake lock can, after a while, just be regularly aborted, which you
+  detect by checking if the exception's name is `'AbortError'`. In this
+  context, `AbortError` is actually not an error in the common sense, but
+  just the way `AbortController` works.
+* The browser can also refuse the request for different reasons, for example,
+  because the battery charge level is too low. In this case, the exception's
+  message will contain more details.
+
+```js
+const wakeLockCheckbox = document.querySelector('#wakeLockCheckbox');
+
+if ('WakeLock' in window) {
+  let wakeLock = null;
+
+  const requestWakeLock = () => {
+    const controller = new AbortController();
+    const signal = controller.signal;
+    window.WakeLock.request('screen', {signal})
+        .catch((e) => {
+          if (e.name === 'AbortError') {
+            wakeLockCheckbox.checked = false;
+            console.log('Wake Lock was aborted');
+          } else {
+            console.error(`${e.name}, ${e.message}`);
+          }
+        });
+    wakeLockCheckbox.checked = true;
+    console.log('Wake Lock is active');
+    return controller;
+  };
+
+  wakeLockCheckbox.addEventListener('change', () => {
+    if (wakeLockCheckbox.checked) {
+      wakeLock = requestWakeLock();
+    } else {
+      wakeLock.abort();
+      wakeLock = null;
+    }
+  });
+}
+```
+
+### The wake lock lifecycle {: #wake-lock-lifecycle }
+
+When you play with the [wake lock demo][demo], you'll notice that wake locks
+are sensitive to [page visibility][page-visibility-api], as well as
+[full screen changes][full-screen-api]. This means that when you minimize a
+tab or window where a wake lock is active, or enter full screen mode, the
+wake lock will automatically abort. If you want your wake lock to be
+re-acquired, you need to listen for the events that either of the APIs emit,
+namely the [`visibilitychange`][visibility-change] event of the
+Page Visibility API, and the [`fullscreenchange`][fullscreen-change] event
+of the Fullscreen API, and then request a new wake lock.
+
+```js
+const handleVisibilityChange = () => {
+  if (wakeLock !== null && document.visibilityState === 'visible') {
+    wakeLock = requestWakeLock();
+  }
+};
+
+document.addEventListener('visibilitychange', handleVisibilityChange);
+document.addEventListener('fullscreenchange', handleVisibilityChange);
+```
+
+## Best Practices {: #best-practices }
+
+The approach you take depends on the needs of your app. However, you should use
```

Comment:
'The approach you take' 

This phrases feels like it's just hanging in mid air. It usually follows or is followed by a listing of the approaches. For example: 'The approach you take to using the Wake Lock API...' 

'However' to 'Regardless'. The Websters definition of 'however' is 'on the other hand'. That doesn't really fit.

Reply:
Can you provide a suggestion on how to fix please?

Reply:
'However' to 'Regardless' was the suggestion.